packages/jira-adapter/src/change_validators/workflowsV2/status_mappings.ts

```diff
@@ -151,6 +153,19 @@ const createStatusMappingStructure = (statusMappings: StatusMapping): string =>
       ]
     },`
 
+const getProjectIssueTypes = async (
+  projectRef: ReferenceExpression,
+  elementsSource:ReadOnlyElementsSource
+): Promise<ReferenceExpression[]> => {
+  const issueTypeSchemaRef = (await projectRef.getResolvedValue(elementsSource)).value?.issueTypeScheme
```

Comment:
the project references are created in this filter with `value` in them, right? so `projectRef.value` is already the project instance, without the need to call `getResolvedValue`.

```suggestion
  const issueTypeSchemaRef = projectRef.value.value.issueTypeScheme
```